cognite/client/data_classes/data_modeling/instances.py

```diff
@@ -64,24 +70,16 @@ class NodeOrEdgeData:
     def load(cls, data: dict) -> NodeOrEdgeData:
         return cls(**convert_all_keys_to_snake_case(data))
```

Comment:
Seems like this fails to load the source into the appropriate data structures? It needs to create a ContainerId or ViewId class. Let's add some unit tests here.